docs/sources/mimir/manage/tools/mimirtool.md

```diff
@@ -799,9 +806,9 @@ mimirtool analyze rule-file <file>
 
 ##### Configuration
 
-| Environment variable | Flag       | Description                                                             |
-| -------------------- | ---------- | ----------------------------------------------------------------------- |
-| -                    | `--output` | Sets the output file path, which by default is `metrics-in-ruler.json`. |
+| Environment variable | Flag                              | Description                                                             |
+| -------------------- | --------------------------------- | ----------------------------------------------------------------------- |
+| -                    | `--output`                        | Sets the output file path, which by default is `metrics-in-ruler.json`. |
```

Comment:
leftover from an earlier draft where the flag was set on this subcommand. will revert.